drivers/serial/uart_sam0.c

```diff
@@ -90,6 +90,29 @@ static void wait_synchronization(SercomUsart *const usart)
 #endif
 }
 
+static int uart_sam0_get_baudrate(SercomUsart *const usart, u32_t *baudrate,
+				  u32_t clk_freq_hz)
+{
+/*
+ * Because the driver uses a baud rate generator in asynchronous mode real baud
+ * rate might be slightly different than the requested nominal baud rate so
+ * uart_sam0_get_baudrate will not necessarily return the same result as what
+ * was programmed with uart_sam0_set_baudrate.
+ */
+	u64_t tmp;
+	u16_t baud;
+
+	baud = usart->BAUD.reg;
+
+	tmp = 65536 - baud;
```

Comment:
Why is this hard-coded at 65536 here? Would it be better to define a local variable with this value? A brief description would help at least.

Reply:
I followed the lead of the already defined `uart_sam0_set_baudrate` function. I was considering turning it into a variable but then I'd need to touch the set_baudrate function as well even though my changes are not related to it. What's the usual approach in a situation like this? I'm assuming that in-file consistency matters (so both get_baudrate and set_baudrate should do the same thing with the magic number) - in that case should one stick with the already-present, ugly, hard-coded number or define a variable and change the rest of the file which wouldn't normally be touched given the functionality change?